corehq/apps/locations/models.py

```diff
@@ -616,17 +629,13 @@ def _key_bounds(self):
     @property
     def descendants(self):
```

Comment:
Can you elaborate?  It's a method on the couch locations class, so my expectation would be that it would return couch locations.  Are you thinking about the future when this will be a method on the `SQLLocations` model?